terraform/applications/discourse/cloudwatch-logs-to-papertrail.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_s3_bucket" "logs_to_papertrail" {
+  acl    = "private"
+  bucket = "logs-to-papertrail-lambda-code"
+}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_lambda_function" "logs_to_papertrail" {
+  # The sourced code of this function is in https://github.com/Signiant/PaperWatch
+  description   = "Sends log from a Cloudwatch subscription to Papertrail"
+  function_name = "arn:aws:lambda:${local.workspace.region}:${data.aws_caller_identity.current.account_id}:function:cloudwatch-to-papertrail"
+  handler       = "src/lambda.handler"
+  memory_size   = "128"
+  role          = aws_iam_role.logs_to_papertrail.arn
+  runtime       = "nodejs10.x"
```

Comment:
I think this version of node is EOL. Would be nice to upgrade or add a ticket/`TODO` here to upgrade at some point.

Reply:
there is an upgrade ticket already.

Reply:
1 step ahead of me! Sounds good